include/openPMD/Series.hpp

```diff
@@ -20,6 +20,10 @@
  */
 #pragma once
 
+// IWYU pragma: private, include <openPMD/openPMD.hpp>
+// IWYU pragma: friend ".*openPMD/.*"
```

Comment:
that's not enough, e.g. everything in `src/` such as `Iteration.cpp` would not be matched